src/Components/Home.js

```diff
@@ -27,6 +27,7 @@ function Home() {
     <div>
       {list.map((user, key) => (
         <Avatar
+          style={{ marginRight: '10px' }}
```

Comment:
Hey ! Nice to see you adding contributions but we are using Prime React so I don't think we need to add custom css. Please have a look at [Prime React](https://www.primefaces.org/primereact/) you will find some way to add margin with the help of classNames.

Still I will ask @eddiejaoude  @nhcarrigan @Cahllagerfeld  to review this once

Reply:
Yeah, it works that way too. I have another idea to represent the profiles with their name with some white badges.

Reply:
Hey @rohan-kulkarni-25 Thanks for the suggestion, I will create a new PR.